lib/gems/pending/util/mount/miq_swift_session.rb

```diff
@@ -0,0 +1,120 @@
+require 'util/mount/miq_generic_mount_session'
+require 'excon'
+
+class MiqSwiftSession < MiqGenericMountSession
```

Comment:
I am going be brutally honest here, and say this is going to have to completely change if we are going through with my changes.  I have already had the changes to the S3 code in place for a few weeks now (been blocked on lack of review for a while now):

https://github.com/ManageIQ/manageiq-gems-pending/pull/361
https://github.com/ManageIQ/manageiq/pull/17894

So either:

1.  We are either completely dropping my RFE
2.  We are going to have to re-write this as well.